OpenTreeMap/Podfile

```diff
@@ -0,0 +1,8 @@
+workspace '../opentreemap.xcworkspace'
```

Comment:
For me `pod install` makes a similar change, but then building causes a linker error. I fixed that error for the PR by removing that reference.

Does it build for you with your updated project file??